LiteCore/tests/LiteCoreTest.cc

```diff
@@ -50,6 +50,20 @@ string TestFixture::sFixturesDir = "../LiteCore/tests/data/";
 string TestFixture::sFixturesDir = "LiteCore/tests/data/";
 #endif
 
+static FilePath GetTempDirectory() {
```

Comment:
This is such gobblty good that it seems a shame to say it twice...

Reply:
Not quite following what you mean

Reply:
Sorry.  It looks like C4Test.cc and LiteCoreTest.cc have almost identical code...

Reply:
There are two distinct testing programs here, and they each have a copy of the code.  Maybe the testing programs need some reworking but that will go down the list.

Reply:
:+1: